internal/pkg/manifest/worker_svc.go

```diff
@@ -206,6 +208,14 @@ func NewWorkerService(props WorkerServiceProps) *WorkerService {
 		svc.WorkerServiceConfig.TaskConfig.CPU = aws.Int(MinWindowsTaskCPU)
 		svc.WorkerServiceConfig.TaskConfig.Memory = aws.Int(MinWindowsTaskMemory)
 	}
+	if len(props.Topics) > 0 {
+		for idx, topic := range props.Topics {
+			if strings.Contains(aws.StringValue(topic.Name), ".fifo") {
+				props.Topics[idx].Name = aws.String(strings.TrimSuffix(aws.StringValue(topic.Name), ".fifo"))
+				svc.WorkerServiceConfig.Subscribe.Queue.FIFO.Enable = aws.Bool(true)
+			}
+		}
+	}
```

Comment:
Can we move this chunk into a separate private function? So that i'll be clearer that we are checking the FIFO.

```go
svc.WorkerServiceConfig.Subscribe.Queue.FIFO.Enable = aws.Bool(fifoEnabled(props.Topics))
```

something like that.